# fix installation directory for gconf schema
# install schema on startup
Index: polly-0.93.12/setup.py
===================================================================
--- polly-0.93.12.orig/setup.py	2016-05-28 02:34:25.001903539 +0200
+++ polly-0.93.12/setup.py	2016-05-28 02:34:25.001903539 +0200
@@ -73,7 +73,7 @@
 
         data_files.append((directory, files))
 
-data_files.append(('/etc/gconf/schemas', ['polly.schemas']))
+data_files.append(('/usr/share/gconf/schemas', ['polly.schemas']))
 
 
 setup(name=b'Polly',
Index: polly-0.93.12/bin/polly
===================================================================
--- polly-0.93.12.orig/bin/polly	2016-05-27 19:00:34.540975000 +0200
+++ polly-0.93.12/bin/polly	2016-05-28 02:55:05.564305973 +0200
@@ -141,6 +141,11 @@
 
 print COPYRIGHT.replace(u'\u00A9', u'(C)')
 
+import subprocess
+cmd = ['gconftool-2', '--install-schema-file=/usr/share/gconf/schemas/polly.schemas']
+proc = subprocess.Popen(cmd, stderr=subprocess.PIPE, stdout=subprocess.PIPE)
+stdout, stderr = proc.communicate()
+assert proc.returncode == 0, stderr
 
 # NOTE: PyXDG inserts XDG_DATA_HOME in the beginning of XDG_DATA_DIRS.
 # This is convenient, but not required by the standard. I leave this
